kedro/templates/project/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/settings.py

```diff
@@ -1,40 +1,41 @@
 """Project settings."""
```

Comment:
I would love to see us elaborate on what `settings.py` is versus `pyproject.toml` and `conf/ `

Basically this image:
![image](https://user-images.githubusercontent.com/35801847/153909111-1a12eac0-3a1e-42c3-a428-5bae710576b9.png)